ndsl/dsl/gt4py_utils.py

```diff
@@ -154,6 +157,9 @@ def make_storage_data(
         data = _make_storage_data_2d(
             data, shape, start, dummy, axis, read_only, backend=backend
         )
+    elif n_dims == 4:
+        
+        data = _make_storage_data_4d(data, shape, start, backend=backend)
```

Comment:
Remove the empty line.

I may have missed it, but there is no test to ensure ndims <=4.  I'm guessing the gt4py.storage.from_array function will crash at that point since data is undefined.